src/pages/blog/2022-04-25-ako-na-monero-node-na-raspberry-pi.md

```diff
@@ -0,0 +1,61 @@
+- jednodoskový počítač: je lacný a má nízku spotrebu energie. Prehľad podporovaných HW [tu](https://github.com/monero-ecosystem/PiNode-XMR/wiki/Hardware#hardware), najrozšírenejšie a asi najľahšie zohnateľné je Raspberry Pi (ja som použila model 4B s 4GB RAM), autor PiNode odporúča Odroid s podporou AES pre rýchlejšie de/šifrovanie a Rock64 pre najlepší pomer cena/výkon.
+- na uloženie blockchainu 256GB MicroSD kartu alebo lepšie externý disk, odkiaľ sa môže bootovať aj OS. Ja som použila SSD Samsung T5 (moje Pi s OS a kompletným Monero blockchainom momentálne zaberá takmer 140 GB).
+
+## Postup
+Stačí postupovať podľa oficiálneho manuálu, podrobný postup od inštalácie OS na Pi aj s tipmi na riešenie problémov nájdete v tomto [videotutoriáli](https://www.youtube.com/watch?v=riK8t_4llXw). Tu sú hlavné body so zameraním na Tor:
+1. je potrebné začať s čerstvou inštaláciou Raspberry Pi OS Lite a defaultným userom pi https://www.raspberrypi.com/software
+2. samotná inštalácia PiNode spočíva v spustení jedného skriptu: https://github.com/monero-ecosystem/PiNode-XMR/wiki/Manual#raspberry-pi, všetky ďalšie nastavenia a úkony sa robia pomocou webového rozhrania alebo systémového menu
+3. v systémovom menu nezabudni zmeniť heslo pre pinodexmr usera a prípadne nastaviť prístupové údaje pre RPC spojenie: https://github.com/monero-ecosystem/PiNode-XMR/wiki/Manual#master-login-password
+4. nainštaluj Tor (v menu časť Extra Network Tools): https://github.com/monero-ecosystem/PiNode-XMR/wiki/Manual#install-tor
+5. na webovom rozhraní (stránka Node Control) si pridaj Tor seed peerov: https://monerodocs.org/infrastructure/tor-onion-p2p-seed-nodes a spusti node v želanom móde: https://github.com/monero-ecosystem/PiNode-XMR/wiki/Manual#anonymity-nodes. Verejnú onion adresu z tejto obrazovky použiješ na pripojenie peňaženky (pozn.: ďalšou výhodou nodu s Torom je, že netreba nastavovať presmerovanie portov na routeri).
+6. ak všetko fičí - na stránke Node Status je vybraný mód active (running), node je Busy Syncing: true a blocky pribúdajú, treba počkať na stiahnutie celého blockchainu (v mojom prípade to trvalo asi 9 dní) a potom môžeš už aj posielať transakcie. Po prvotnej synchronizácii nie je beh lokálneho nodu záťažou pre procesor a sieť.
+
+## Nastavenie peňaženky Monerujo
+1. nainštaluj si [Orbot](https://guardianproject.info/apps/org.torproject.android), ktorý umožňuje používať mobilné aplikácie so sieťou Tor
+2. v Orbot zapni **VPN Mode** a kolieskom v sekcii **Tor-Enabled Apps** nastav Monerujo (a ďalšie appky, ktoré chceš používať cez Tor), aby to vyzeralo takto:
+![](/assets/monero-node-orbot.png)
+Orbot zavri.
+3. spusti [Monerujo](https://www.monerujo.io) (Orbot sa na pozadí sám naštartuje), ťukni na ikonku pred názvom remote nodu, zmení sa na cibuľu Toru:
+<p align="center">
+  <img alt="clear" src="/assets/monero-node-01.jpg" width="45%">
```

Comment:
Poprosim ta pridat tieto obrazky cez markdown (ako je napr. [vyssie](https://github.com/ParalelnaPolisKE/paralelnapoliskosice.sk/pull/154/files#diff-e621f87b103b1ac03ee79214adafad04e5dcce799d71e2c0ecd0ea3f5821d6ddR39)). Nasledne budu spravne spracovane a optimalizovane.

Je mozne, ze to vnutri `p` tagu nepojde, v tom pripade je moznost tie dva obrazky spojit do jedneho.

Rovnako aj dvojica obrazkov nizsie. Vdaka :)

Reply:
zmenene, naucila som sa spojit v GIMPe :)